modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -780,8 +780,24 @@ bool CvCapture_FFMPEG::open( const char* _filename )
     ic->interrupt_callback.opaque = &interrupt_metadata;
 #endif
 
-#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(52, 111, 0)
+#if LIBAVFORMAT_BUILD > CALC_FFMPEG_VERSION(51,11,0)
```

Comment:
Why is this change here?
- Condition must be ">=". ">" should not be used at all for version checks.
- Perhaps we should check for [53.4.0](https://github.com/FFmpeg/FFmpeg/blob/c79e7534712fa9e95a9ddde2c5e8c6815f5f2f6f/doc/APIchanges#L1986) (avformat_open_input)
- These change must be propagated to many places with "dict" (av_dict_free / declaration / etc)